integ-test/src/test/java/org/opensearch/sql/legacy/JdbcTestIT.java

```diff
@@ -78,8 +78,6 @@ public void testDateTimeInQuery() {
         equalTo("17-08-2014"));
   }
 
-  @Ignore("flaky test, trigger resource not enough exception. "
-      + "ORDER BY all_client/10 can't be pushed down ")
```

Comment:
So we confirm the ORDER BY can be pushed down and thus won't trigger circuit breaker now?

Reply:
I'm not sure that it will never trigger circuit breaker. From what I've observed from multiple GitHub Actions runs and running it locally, they have the same results when the fetch_size specifications are the same. 

I can revert this change if you are not confident that it won't trigger circuit breaker.

Reply:
yeah, from what it says it seems disabled due to circuit break triggered sometimes. We can enable it if we confirm the pushdown works now. Otherwise we may want to leave it as is.